libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -3,10 +3,24 @@ import { CommonModule } from '@angular/common';
 import { HomePageComponent } from './pages/home-page/home-page.component';
 import { HomeHeroSectionComponent } from './components/home-hero-section/home-hero-section.component';
 import { LayoutModule } from '@elewa-group/elements/layout';
+import { JobsSectionComponent } from './components/jobs-section/jobs-section.component';
+import { Routes, RouterModule } from '@angular/router';
```

Comment:
order of imports is not correct, should be. 

import { Routes, RouterModule } from '@angular/router';

import { LayoutModule } from '@elewa-group/elements/layout';

import { JobsSectionComponent } from './components/jobs-section/jobs-section.component';
import { HomeHeroSectionComponent } from './components/home-hero-section/home-hero-section.component';

import { HomePageComponent } from './pages/home-page/home-page.component';